UOP1_Project/.gitignore

```diff
@@ -23,6 +23,9 @@
 .vs/
 .vscode/
 
+# Idea cache directory
+.idea/
```

Comment:
I use [Rider](https://www.jetbrains.com/rider) so I dont want store `.idea` preferences folder

Reply:
This doesn't belong to this PR though. Can you remove it and put it in a different one?

Reply:
Done